src/google.ts

```diff
@@ -96,9 +104,32 @@ export function createGoogleClient(
     }
   }
 
+  /**
+   *
+   * A function to add a member to the beta group.
```

Comment:
```suggestion
   * Adds a member to the beta group.
```

in general we shouldn't say it's a function or a class (that is obvious), just what it does